code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -87,9 +87,10 @@
 
 /obj/item/weapon/reagent_containers/hypospray/autoinjector/attack(mob/M as mob, mob/user as mob)
 	..()
-//	if(reagents.total_volume <= 0) //Prevents autoinjectors to be refilled.
-//		flags &= ~OPENCONTAINER
+	if(reagents.total_volume <= 0) //Prevents autoinjectors to be refilled after use.
+		flags &= ~OPENCONTAINER
 	update_icon()
+	return
```

Comment:
Why

Reply:
I see code I copypaste code
https://github.com/vgstation-coders/vgstation13/pull/18725/files

Reply:
Well, remove this line

Reply:
done